notary-signer/Dockerfile

```diff
@@ -10,7 +10,7 @@ WORKDIR ${INSTALLDIR}
 
 RUN set -eux; \
     apk add --no-cache --virtual build-deps git go make musl-dev; \
-    go version | grep 'go1.12.8 '; \
+    go version | grep 'go1.12.10'; \
```

Comment:
oh, hm. wondering if the space should've been left; I thought it was a copy/paste error on my side (it did build succesfully though).

Let me double check 😊

Reply:
its unlikely that there will be a go1.12.100 released so its probably safe...

Reply:
the output is eg `go version go1.13.1 darwin/amd64` so the space was intentional...

Reply:
Yeah, but it could match `go1.12.0-beta1` (not sure if they use that format); https://github.com/docker/notary-official-images/pull/19

Reply:
They don't use hyphens or a `.0` officially, so it'd be `go1.14beta1`, `go1.12`, `go1.12.10`, etc.

Reply:
ah, yes

to be `go mod` compliant 😉😂